content/_img-sources/amm-concept-swap.uxf

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These three diagram files are currently unused. The hope is for a follow-up to add diagrams based on/inspired by these. Here's what the rough draft of this one looks like:
![amm-concept-swap](https://user-images.githubusercontent.com/7515597/207204026-91764fbf-70e3-421b-8a14-7bddfb625868.png)